gqueries/modules/security_of_supply/loss_of_load_expectation.gql

```diff
@@ -10,6 +10,7 @@
       energy_power_wind_turbine_coastal,
       energy_power_wind_turbine_inland,
       energy_power_wind_turbine_offshore,
+      energy_power_hybrid_wind_turbine_offshore,
```

Comment:
Nice catch. It seems like more nodes might be missing here. There is already an issue for this: https://github.com/quintel/etsource/issues/2949

Reply:
Cool, I've added the issue to the Modeller sprint backlog.